AjobK/Athena
# 194 profile info

closes #194

 **What is it supposed to do**
This component is supposed to display a profile description and you should be able to edit this as well. 

 **How can we test your branch**
- login
- you should see your profile description
- click edit
- edit the description
- save the description
- if you refresh the profile page the description should be updated


 **What browser have been checked**
